docs/guides/data_viewers/tableau.md

```diff
@@ -29,35 +28,100 @@ CREATE VIEW h2oai AS (
+than the provided Postgres dialect.
+
+The documentation on how to install and use the connector is in its repository,
+but essentially you will need the `duckdb_jdbc.taco` file from the `duckdb-taco/packaged-connector` directory.
+The current version of the Taco is not signed, you will need to launch Tableau with signature validation disabled.
+(Despite what the Tableau documentation days, the real security risk is in the JDBC driver code,
+not the small amount of JavaScript in the Taco.)
+
+## Server (Online)
+
+On Linux, copy the Taco file to `/opt/tableau/connectors`.
+On Windows, copy the Taco file to `C:\Program Files\Tableau\Connectors`.
+Then issue the commands to disable signature validation:
+
+```sh
+$ tsm configuration set -k native_api.disable_verify_connector_plugin_signature -v true
+$ tsm pending-changes apply
+```
+The last command will restart the server with the new settings.
 
-## MacOS
+## MacOS Desktop
 
-As of this writing, the MacOS DuckDB ODBC Driver seems to crash 
-the Tableau Protocol Server (a sub-process used to isolate drivers from the main application).
-This means you should use the JDBC driver for now.
+Copy the Taco file to the `/Users/[MacOS User]/Documents/My Tableau Repository/Connectors` folder.
+Then launch Tableau Desktop from the Terminal with the the command line argument to disable signature validation:
 
-### Install the DuckDB JDBC Driver:
+```sh
+$ /Applications/Tableau\ Desktop\ <year>.<quarter>.app/Contents/MacOS/Tableau -DDisableVerifyConnectorPluginSignature=true
+```
 
-1. [Download](https://github.com/duckdb/duckdb/releases/download/v{{ site.currentduckdbversion }}/duckdb_jdbc-osx-universal.jar) the DuckDB JDBC driver. This is a file called `duckdb_jdbc-osx-universal.jar`. Make sure this is from the same build as the version of DuckDB used to create the database file!
-2. Follow the [Tableau directions](https://help.tableau.com/current/pro/desktop/en-us/jdbc_tableau.htm) and copy this file to either `/Library/JDBC` (for access by all users) or `~/Library/JDBC` (for access by your login alone) (e.g., `cp ~/Downloads/duckdb_jdbc-osx-universal.jar ~/Library/JDBC`)
+You can also package this up with AppleScript by using the following script:
 
-### Connect to your data
+```
+do shell script "\"/Applications/Tableau Desktop 2023.2.app/Contents/MacOS/Tableau\" -DDisableVerifyConnectorPluginSignature=true"
+quit
+```
 
-3. Create a  DuckDB file containing your views and/or data.
-4. Launch Tableau
-5. Under Connect > To a Server > More... click on "Other Databases (JDBC)"  This will bring up the connection dialogue box. For the URL, enter `jdbc:duckdb:/User/username/path/to/database.db`. For the Dialect, choose `PostgreSQL`. the rest of the fields can be ignored:
+Create this file with Script Editor and save it as a packaged application. You can then double-click it to launch Tableau. You will need to change the application name when you get upgrades.
```

Comment:
@hawkfish How does one create this file with script editor? Is Script Editor something that comes with Mac or does it need to be installed separately?